### ISSUE TITLE
Adding Stripe to BeYourMarket

### ISSUE THREAD

**Issue by Alex77778** [beyourmarket/beyourmarket#60]
Has anyone else had any troubles integrating Stripe to BeYourMarket? I've followed everything in the documentation but nothing seems to be working. None of the three plugins (Stripe, Google Analytics, Disqus) are loaded into my Manage Plugins page on my Admin Panel like they are on the Demo Admin Panel. If anyone knows how to set up Stripe please let me know.

**Comment by mwillebrands** [comment]
Do you have any errors in your error log? Are you running the apllication in full Trust mode?

**Comment by Alex77778** [comment]
No errors right now and I don't believe so, I seem to be having troubles with BeYourMarket.Web detecting the Stripe plugin (detailed here: https://beyourmarket.atlassian.net/wiki/display/BYM/Create+a+BeYourMarket+Plugin ). If/when you set up BeYourMarket were the plugins already displayed in the Manage Plugins page? It seems everything on that Create a Plugin page I linked had already been done when I cloned the git so I guess I'm just confused on how to make everything work.
